cookie.ts

```diff
@@ -29,6 +30,7 @@ export function formatCookieString (key : string, value : any, options : CookieO
         domain: options.domain || false,
         path: options.path || "/",
         secure: options.secure || (window.location.protocol === "https:"),
+        sameSite: options.sameSite || "strict",
```

Comment:
`null || "strict"` will always result to `"strict"`

You need to do `options.sameSite !== undefined ? options.sameSite : "strict"`